lib/fluent/supervisor.rb

```diff
@@ -1104,6 +1104,11 @@ def build_spawn_command
         fluentd_spawn_cmd << '-Eascii-8bit:ascii-8bit'
       end
 
+      if @system_config.enable_jit
+        $log.info "enable Ruby JIT for workers"
+        fluentd_spawn_cmd << '--jit'
```

Comment:
It may be obvious, but it is better to mention --jit (YJIT since Ruby 3.1) because the default meaning is changed.

Reply:
Added. Thanks.